includes/data-stores/class-wc-customer-data-store.php

```diff
@@ -114,7 +114,7 @@ public function create( &$customer ) {
 		wp_update_user( apply_filters( 'woocommerce_update_customer_args', array(
 			'ID'           => $customer->get_id(),
 			'role'         => $customer->get_role(),
-			'display_name' => $customer->get_first_name() . ' ' . $customer->get_last_name(),
+			'display_name' => sprintf( __( '%1$s %2$s', 'display name', 'woocommerce' ), $customer->get_first_name(), $customer->get_last_name() ),
```

Comment:
According to Scrutinizer this code potentially leaking two bugs for each $customer->get_ functions, `The method get_first_name() does not seem to exist on object<WC_Data>` (tough I reuse and modify the code a little). What do you think if I change this `$customer->get_display_name()`?